frontend/src/components/Components/Search/index.js

```
@@ -0,0 +1,70 @@
+import get from "lodash/get";
```

ESLint

[error] Extra semicolon.

(semi)